perfkitbenchmarker/linux_benchmarks/memcached_ycsb_benchmark.py

```diff
@@ -69,6 +69,9 @@
     installation. Specify the number of YCSB client VMs with
     --ycsb_client_vms and the number of YCSB server VMS with
     --num_vms.
+  flags:
+    ycsb_client_vms: 1
```

Comment:
It's a [YCSB flag](https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/blob/master/perfkitbenchmarker/linux_packages/ycsb.py#L101)